.github/workflows/ci.yml

```diff
@@ -197,3 +203,55 @@ jobs:
           ENDPOINT_URL: '${{ env.INTEGRATION_WEBHOOK_URL }}/webhook'
         run: |-
           go test github.com/abcxyz/github-metrics-aggregator/integration -timeout=15m
+
+  deployment_webhook_autopush:
+    runs-on: 'ubuntu-latest'
```

Comment:
There should be an `if: ${{ github.event_name == 'push' }}` here, we should only deploy to autopush on merge